modelskill/connection.py

```diff
@@ -240,6 +240,10 @@ def _extract_from_models(obs, mod: List[protocols.ModelResult]) -> List[pd.DataF
             mr = mr.extract(obs)
 
         df = mr.data
+
+        # TODO is this robust enough?
+        old_item = df.columns.values[-1]
```

Comment:
Let's do it like this for now and then revisit when we allow arbitrary aux items